plugins/docker/docker_

```diff
@@ -107,18 +243,39 @@ def print_containers_cpu(client):
     for container, stats in parallel_container_stats(client):
         cpu_count = len(stats["cpu_stats"]["cpu_usage"]["percpu_usage"])
         cpu_percent = 0.0
-        cpu_delta = float(stats["cpu_stats"]["cpu_usage"]["total_usage"]) \
-            - float(stats["precpu_stats"]["cpu_usage"]["total_usage"])
-        system_delta = float(stats["cpu_stats"]["system_cpu_usage"]) \
-            - float(stats["precpu_stats"]["system_cpu_usage"])
+        cpu_delta = float(stats["cpu_stats"]["cpu_usage"]["total_usage"])
+        - float(stats["precpu_stats"]["cpu_usage"]["total_usage"])
```

Comment:
I think, you need to add parenthesis around the two lines - otherwise the second line is just interpreted as a (negative) number without any impact on the calculation above.

Reply:
Ah, right! Yeah, I thought it was weird that it deindented it... Somehow I lost the `\`, but parens will be less brittle.